modules/ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_{context}"]
+= Configuring the hub cluster to use unauthenticated registries
+
+You can configure the hub cluster to use unauthenticated registries.
+
+.Procedure
+
+. Edit the `AgentServiceConfig` CR to include the `unauthenticatedRegistries` parameter:
+
+[source,terminal]
+----
+$ apiVersion: agent-install.openshift.io/v1beta1
+kind: AgentServiceConfig
+metadata:
+  name: agent
+spec:
+  unauthenticatedRegistries:
+  - example.registry.com
+  - example.registry2.com
+  ...
+----
+
+Unauthenticated registries are listed under `spec.unauthenticatedRegistries` in the `AgentServiceConfig` resource.
+Any registry on this list is not required to have an entry in the pull secret validated by the `assisted-service`.
```

Comment:
I would clarify that it is the pull secret created for the spoke cluster

Reply:
Updated as suggested.